src/frameworks/tide.rs

```diff
@@ -14,12 +14,12 @@ use crate::{
 #[derive(Clone)]
 struct State {
     repo: Arc<MongoRepo>,
-    runtime: Arc<Runtime>,
+    runtime: Handle,
 }
 
 pub async fn start(db: Database) -> Result<(), Box<dyn std::error::Error>> {
-    // Create a Tokio runtime for MongoDB operations
-    let runtime = Arc::new(Runtime::new()?);
+    // Get a handle to the Tokio runtime for MongoDB operations
+    let runtime = Handle::current();
```

Comment:
Because the app was started in a `main` fn that has `#[tokio::main]`, there's an implicit Tokio runtime created already, and that's the one the `Client` was created on.  Creating a client in one runtime and then running code using it in a second runtime instance is one of those things that accidentally works most of the time and then causes _really_ weird errors in unexpected places.

Happily, Tokio's `Handle` type is internally refcounted and freely `Clone`-able so we can also drop the `Arc` wrapper :)